qiskit_bot.yaml

```
@@ -267,10 +267,13 @@ notifications:
     - "@jyu00"
     - "@ElePT"
     - "@beckykd"
-  "docs/migration-guides/sessions":
+  "docs/guides/job-limits":
     - "@jyu00"
     - "@ElePT"
     - "@beckykd"
+  "docs/migration-guides/sessions":
+    - "@blakejohnson"
+    - "@beckykd"
```

```suggestion
  "docs/guides/job-limits":
    - "@jyu00"
    - "@ElePT"
    - "@beckykd"
  "docs/migration-guides/sessions":
    - "@blakejohnson"
    - "@beckykd"
```
I think the file names got switched here?